ivc/src/ivc/columns.rs

```diff
@@ -284,78 +293,90 @@ impl ColumnIndexer for IVCColumn {
     // This should be
     //   const N_COL: usize = std::cmp::max(IVCPoseidonColumn::N_COL, FECColumn::N_COL);
     // which is runtime-only expression..?
-    const N_COL: usize = IVCPoseidonColumn::N_COL;
+    // 333 is not enough
+    const N_COL: usize = 400;
 
     fn to_column(self) -> Column {
         match self {
+            IVCColumn::BlockSel(i) => {
+                assert!(i < 2 * N_BLOCKS);
+                Column::FixedSelector(i)
+            }
+
             IVCColumn::Block1Input(i) => {
                 assert!(i < 2 * N_LIMBS_SMALL);
-                Column::Relation(i)
+                Column::Relation(N_BLOCKS + i)
```

Comment:
What's in `Column::Relation(0..N_BLOCKS)`?

Reply:
Will fix!